src/utils/ExtrusionJunction.h

```diff
@@ -47,6 +47,14 @@ struct ExtrusionJunction
     ExtrusionJunction(const Point p, const coord_t w, const coord_t perimeter_index, const size_t region_id = 0);
 
     bool operator==(const ExtrusionJunction& other) const;
+
+    void InternalSwap(ExtrusionJunction* other);
+
+    void Swap(ExtrusionJunction* other);
+    friend void swap(ExtrusionJunction& a, ExtrusionJunction& b)
```

Comment:
newline between functions
```suggestion

    friend void swap(ExtrusionJunction& a, ExtrusionJunction& b)
```

Reply:
Agreed, but I removed this code. Swap on POD-members like the ones in ExtrusionJunction will end up moving the data anyway. No need to specify that to swap this class, you need to swap each of its members.